### PR TITLE
Update ReactNativeWeb example

### PR DESCRIPTION
**Pull request by alloy**
For some reason the button label doesn’t v-align center in Safari, but probably an easy fix. Looks like this in Chrome:

screen shot 2017-01-14 at 12 58 14

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -81,5 +81,5 @@ app.use(function (req, res, next) {
 });
 
 app.listen(3000, () => {
-  console.log("✨ Opened on http://localhost:3000"); // tslint:disable-line
+  console.log("✨  Listening on http://localhost:3000"); // tslint:disable-line
```
Comment (orta) [inline index.js:84]: weird, I thought I did that

**Comment by alloy** [comment]
This can now be reviewed/merged.

**Comment by alloy** [comment]
Going to try https://github.com/necolas/react-native-web/pull/304 to see if it cleans up the inline CSS styling soup and if it makes rehydration possible out-of-the-box.

File: app/containers/react-native-web/artist/grid.tsx [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+  }
+}
+
+interface Styles {
+  container: ViewStyle,
+  section: ViewStyle,
+  spinner: ViewStyle,
+}
+
+const styles = StyleSheet.create<Styles>({
```
Comment (orta) [inline app/containers/react-native-web/artist/grid.tsx:312]: this is a nice idea, I assume then it does correct autocompletion inside the children?

File: app/containers/react-native-web/artist/header.tsx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+  static propTypes: Object = {
+    artist: React.PropTypes.shape({
+      name: React.PropTypes.string,
+      nationality: React.PropTypes.string,
+      birthday: React.PropTypes.string,
+      counts: React.PropTypes.shape({
+        follows: React.PropTypes.number,
+      }),
+    }),
+  };
```
Comment (orta) [inline app/containers/react-native-web/artist/header.tsx]: We shouldn't need to do these, I assume you just ported it over

File: types/react-native-web.d.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+/// <reference path="./react-native.d.ts" />
+
+declare module "react-native-web" {
+  export = React
+}
```
Comment (orta) [inline types/react-native-web.d.ts:5]: hah, nice one

for those not following along, this basically redirects react-native-web to re-use the imports from react-native

**Review by orta — APPROVED** [review]
Feels 👍

**Comment by orta** [comment]
Feels like writing React Native

File: app/containers/react-native-web/artist/grid.tsx (hunk 2 above)
Comment (alloy) [reply app/containers/react-native-web/artist/grid.tsx:312]: Aye, and the objects that come out of `styles.foo` will be typed.

File: app/containers/react-native-web/artist/header.tsx (hunk 3 above)
Comment (alloy) [reply app/containers/react-native-web/artist/header.tsx]: Ah yes, in another place I removed them, though, just lost track I guess.

FYI There’s one difference, which is that React’s prop validation happens at runtime.

**Comment by alloy** [comment]
Yup, it does so very much. In some cases it’s unfortunately buried under purely TS related changes.

File: app/containers/react-native-web/artist/header.tsx (hunk 3 above)
Comment (alloy) [reply app/containers/react-native-web/artist/header.tsx]: I now see I didn’t type the artist either, will update.

**Comment by alloy** [comment]
Alright, done, merging

**Pull request merged**